helm-chart/renku-core/templates/deployment-nginx.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "renku-core.fullname" $ }}-nginx
+  labels:
+    app.kubernetes.io/name: {{ include "renku-core.fullname" $ }}-nginx
+spec:
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: {{ include "renku-core.fullname" $ }}-nginx
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: {{ include "renku-core.fullname" $ }}-nginx
+    spec:
+      containers:
+        - name: nginx
+          image: nginx:alpine
+          imagePullPolicy: IfNotPresent
+          ports:
+          - containerPort: 80
```

Comment:
Listening on port 80 needs some privileges that we'd like to restrict (see above). Just move it to some port higher than 1000 and switch the target port in the k8s service spec